.circleci/src/jobs/check.yml

```diff
@@ -7,4 +7,7 @@ environment:
 shell: /bin/bash -eo pipefail
 steps:
   - checkout
-  - check_circleci_config
\ No newline at end of file
+  - install_app_toolbelt
+  - check_circleci_config
+  - notify:
+      allow-all-branches: true
```

Comment:
This will not send anything to mattermost if this fails, right?

Reply:
no this should be removed, thanks.